babybuddy/management/commands/createuser.py

```diff
@@ -51,6 +51,13 @@ def add_arguments(self, parser):
             dest="password",
             help="Specifies the password for the user. Optional.",
         )
+        parser.add_argument(
```

Comment:
I prefer we abstract away the fact that these are groups at all. From the perspective of a Baby Buddy user I just want to offer the three "types" of users. So instead of a `--groups` flag that could theoretically take _any_ group name lets just have two (exclusive) flags:

- `--read-only`
- `--staff`

If the `--read-only` flag is set the user is added to the `read_only` group and if the `--staff` flag is set the user is made staff and superuser (only one flag should be settable at a time). If neither flag is set the user gets added to the `standard` group.